website/docs/patterns/multi-select-patterns/partials/specifications/specifications.md

```diff
@@ -21,19 +21,19 @@ Within the Multi-select pattern itself, use a 16px horizontal gap between the me
 
 ### In context
 
-Within a larger context, use the same spacing recommendation of 16px vertical gap between the pattern, filter elements, and data set.
+Within a larger context, use a 16px vertical gap between the filter elements, multi-selection pattern, and table.
 
 ![Spacing in context with other elements](/assets/patterns/multi-select-patterns/multi-select-in-context-spacing.png)
 
-## Basic layout
+## Layout
 
 In a basic example of a Table, place the selection pattern above the top-right corner of the Table.
```

Comment:
```suggestion
For a basic Table, place the selection pattern above the top-right corner of the Table.
```

Reply:
Btw, what would a "basic" table be vs. a non-basic table for purposes of these examples?

Reply:
A basic table would be one that doesn't use filtering.

Reply:
Maybe mention that to clarify?